lib/fluent/plugin/buffer/chunk.rb

```diff
@@ -165,6 +166,18 @@ def write_to(io, **kwargs)
         module Decompressable
           include Fluent::Plugin::Compressable
 
+          def bytesize(**kwargs)
```

Comment:
IMO the computing cost to return bytesize of decompressed data is too high, and it's reasonable to omit this feature.
Are there any use case?